src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/util/IndexUtils.kt

```diff
@@ -124,5 +140,36 @@ class IndexUtils {
                 actionListener.onResponse(AcknowledgedResponse(false))
             }
         }
+
+        @OpenForTesting
+        fun checkAndUpdateAliasMapping(
+            alias: String,
+            schemaVersion: Long,
+            mapping: String,
+            clusterState: ClusterState,
+            client: IndicesAdminClient,
+            actionListener: ActionListener<AcknowledgedResponse>
+        ) {
+            clusterState.metadata.hasAlias(alias)
```

Comment:
The result is initialized to the variable so it can be used further down int he else statement without looking it up again.
Removed unneeded hasAlias line though.